### DISCUSSION TITLE
Codesandbox - Twilio Alarm Setup page and Manage Alarms page

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1502] [Office hours] [answered]
Hello friends, 
I've created a [codesanbox prototype](https://codesandbox.io/s/alarms-forked-05-26-2021-80igm) based on my [figma design mock ](https://www.figma.com/file/3GaINOa9Epx7ENTZslfaWa/Alarms-H1-2021?node-id=1890%3A9554). It looks good but not perfect. 

My team doesn't have FE resources so I need your help on few things... 
1. How to add a dynamic graph in the Alarm setup page. 
2. How to add a custom image (Console indicator icon) into the help text. 
3. I've tried to add status indicator(Alarms, Warning and Resolved) into the table but I failed.  
Cheers,

**Comment by SiTaggart** [comment, marked as answer]
Hi @sang-yeo, we'll take a look and see if we can provide any tips before Office Hours, otherwise we'll see you then.

**Comment by sang-yeo** [comment]
👋 I've updated the [codesandbox](https://codesandbox.io/s/alarms-forked-05-26-2021-80igm) mock and few things that I couldn't change by myself.  
1. I would like to remove the Actions column from the Alarms history page
2. I would like to add a toast component interaction when I click 'Save' button from the Create alarm page but it(a toast component) appears in the Manage  alarms page. 
3. I would like to swap the 'garbage can' icon from the Actions column in the Alarms history to 'more' icon 

thanks,
